main.py

```diff
@@ -1,6 +1,43 @@
 """
 Codebreaker
 """
+import cProfile
+def dr_test(x, y, box, code, old_guesses, count):
+    for e in range(x):
+        guess_code = []
+        hint = []
+        print('Pick from R, B, Y, G, O, or P')
+        print('Type quit to end early')
+        for i in range(y):
+            guess = (input(f'Guess the {i + 1} position: ')).upper()
+            if guess == 'QUIT':
+                return print("You ended early, better luck next time")
+            while guess not in set(box):
+                print('Invalid input. Guess must be from R, B, Y, G, O or P')
+                guess = (input(f'Guess the {i + 1} position: ')).upper()
+            guess_code.append(guess)
+        guess_copy = guess_code.copy()
+        code_copy = code.copy()
+        for i, e in enumerate(guess_copy):
+            if guess_copy[i] == code_copy[i]:
+                hint.append('Black')
+            elif guess_copy[i] in code_copy:
+                hint.append('White')
+            else:
+                hint.append('-')
+        old_guesses.update({
```

Comment:
the old syntax where you were writing a dictionary in reverse worked, but it hurt my head. 

I opted for using `.update()`